compatibility.txt

```diff
@@ -3,6 +3,9 @@
 ps_accounts=$("$MODDIR/sqlite" $PS_DATA_PATH "SELECT account FROM ownership" | sort -u | wc -l)
 ps_accounts_final=$((ps_accounts+1))
 
+MAGMOD=/data/adb/modules/Detach
```

Comment:
I would prefere to have common vars at the beginning of the `compatibility.txt` file